modules/themes/comfyTheme.js

```diff
@@ -0,0 +1,370 @@
+const version = "0";
```

Comment:
This should probably be a proper version.

Reply:
Usually `@import` theme modules have a version of 0, but I suppose this could have a proper version considering that it has some extra code in there. Go ahead and change it to 1.0.0, I'm on my phone right now unfortunately.

Reply:
Sure.